src/NHibernate/Util/SerializationHelper.cs

```diff
@@ -1,32 +1,60 @@
-using System;
 using System.IO;
+using System.Runtime.Serialization;
 using System.Runtime.Serialization.Formatters.Binary;
 
 namespace NHibernate.Util
 {
-	public class SerializationHelper
+	public static partial class SerializationHelper
 	{
-		private SerializationHelper()
+		private static readonly BinaryFormatter Formatter = new BinaryFormatter
```

Comment:
It cannot be stored because it is [not thread safe](https://msdn.microsoft.com/en-us/library/system.runtime.serialization.formatters.binary.binaryformatter(v=vs.110).aspx#Anchor_8). It must be created on a per usage basis (or at least put in a thread static variable).

Reply:
Ok